orchagent/flexcounterorch.cpp

```diff
@@ -328,11 +329,41 @@ bool FlexCounterOrch::bake()
     return consumer->addToSync(entries);
 }
 
+static bool isCreateOnlyConfigDbBuffers(Table& deviceMetadataConfigTable)
+{
+    std::string createOnlyConfigDbBuffersValue;
+
+    try
+    {
+        if (deviceMetadataConfigTable.hget("localhost", "create_only_config_db_buffers", createOnlyConfigDbBuffersValue))
+        {
+            if (createOnlyConfigDbBuffersValue == "true")
+            {
+                return true;
+            }
+        }
+    }
+    catch(const std::system_error& e)
+    {
+        SWSS_LOG_ERROR("System error: %s", e.what());
```

Comment:
can you change this to WARNING? for non existent config_db, we don't want an ERR log printed

Reply:
The error statement in the `catch` block will be printed only if the `deviceMetadataConfigTable.hget()` throws an exception.
If the `create_only_config_db_buffers` do not exist in the `config_db` the `deviceMetadataConfigTable.hget()` returns `false`.